share/models/fields.py

```diff
@@ -550,3 +555,37 @@ def bulk_related_objects(self):
         # https://github.com/django/django/blob/master/django/db/models/deletion.py#L151
         # Disable django cascading deletes for this field
         raise AttributeError('This is a dirty hack')
+
+
+class EncryptedJSONField(models.BinaryField):
+    """
+    This field transparently encrypts data in the database. It should probably only be used with PG unless
+    the user takes into account the db specific trade-offs with TextFields.
+    """
+    prefix = b'jwe:::'
+
+    def get_db_prep_value(self, input_json, **kwargs):
+        if input_json:
+            input_json = json.dumps(input_json).encode('utf-8')
+            try:
+                input_json = self.prefix + jwe.encrypt(input_json, SENSITIVE_DATA_KEY)
+
+            except InvalidTag:
```

Comment:
You can remove this entire try block, if `raise` is the only statement in the except it effectively does nothing.